pyanaconda/modules/runtime/user_interface/ui_interface.py

```diff
@@ -61,6 +68,103 @@ def PasswordPolicies(self, policies: Dict[Str, Structure]):
             PasswordPolicy.from_structure_dict(policies)
         )
 
+    @property
+    def DisplayMode(self) -> Str:
+        """The display mode for the installation.
+
+        Possible values are "TUI" and "GUI".
```

Comment:
What about the `cmdline` mode?

Reply:
I've added the cmdline option here, but I don't know if it's completely correct if it's actually non-interactive text.